### ISSUE TITLE
[docsify-dark-mode] When switching to dark mode, the text is hard to read.

### ISSUE THREAD

**Issue by Gaspacchio** [anikethsaha/docsify-plugin#5]
## Problem

If the `themeColor` is set to something dark, say `#1e4691`, switching to the dark mode makes the text unreadable (see screenshot below ⬇)

## Preferred behavior

The text color should be set to something light by default.

## Additional material

![image](https://user-images.githubusercontent.com/43613824/75360151-b491cb80-58b5-11ea-9096-406156a56745.png)

**Comment by anikethsaha** [comment]
Hey ! thanks again for reporting 

can you try with [`textColor` option](https://github.com/anikethsaha/docsify-plugin/tree/master/packages/docsify-dark-mode#configure)

**Comment by Gaspacchio** [comment]
Ooooh, I see, of course! I'm trying this right now.

**Comment by anikethsaha** [comment]
> ! I'm trying this right now.

Yeah sure, just try and let me know if you think it needs to be changed or not. Even you can submit a PR with your changes which you think will fit fine here
